core/src/Streamly/Internal/FileSystem/File.hs

```diff
@@ -204,15 +205,15 @@ usingFile3 = UF.bracketIO before after
 -- /Pre-release/
 --
 {-# INLINABLE putChunk #-}
-putChunk :: FilePath -> Array a -> IO ()
+putChunk :: forall (a :: Type). FilePath -> Array a -> IO ()
```

Comment:
Why is this important? Can we ignore this using `-Wno-`?

Reply:
It is an error, not a warning.